coturn/coturn
# Disable systemd when building on macOS using CMake

systemd is not available on macOS - should be disabled in CMake 

Test plan:
- Run cmake in a clean folder
- Run make
- Observe that compilation error about missing include `<systemd/sd-daemon.h>` is gone
